### PR TITLE
BUG: Correct delvewheel path in Windows module build wheel script

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: scripts/windows_build_module_wheels.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def fixup_wheel(py_envs, filepath, lib_paths:str='', exclude_libs:str=''):
 
     py_env = py_envs[0]
     
-    delve_wheel = os.path.join(ROOT_DIR, "venv-" + py_env, "Scripts", "delvewheel.exe")
+    delve_wheel = os.path.join("C:/P/IPP", "venv-" + py_env, "Scripts", "delvewheel.exe")
```
Comment (tbirdso) [inline scripts/windows_build_module_wheels.py:116]: What value were you previously seeing for `ROOT_DIR`? Are there concerns about relying on a more rigid file system with this change?

`ROOT_DIR` is previously defined at line 13 as the executing directory, is this not typically `C:/P/IPP`?
```py
ROOT_DIR = os.path.abspath(os.getcwd())
```

Comment (thewtex) [reply scripts/windows_build_module_wheels.py:116]: When building modules, the `ROOT_DIR` is the module source directory, which is different from `C:/P/IPP`. There is already a rigid filesystem structure required, which should be improved, but those are (many) other patches :-).

Comment (thewtex) [reply scripts/windows_build_module_wheels.py:116]: After adding this changes to `ITKPythonBuilds-windows.zstd`, the Windows builds here now succeed: https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/123

Comment (tbirdso) [reply scripts/windows_build_module_wheels.py:116]: Gotcha. Thanks for the explanation!

**Review by tbirdso — APPROVED** [review]

**Pull request merged**